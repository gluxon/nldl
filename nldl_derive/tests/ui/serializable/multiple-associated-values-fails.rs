#[derive(nldl::attr::Serialize)]
enum TestEnum {
    #[nla_type(1)]
    Flags(u32, u32)
}

fn main() {}
