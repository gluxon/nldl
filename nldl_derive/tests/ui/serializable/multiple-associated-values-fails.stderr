error: proc-macro derive panicked
 --> tests/ui/serializable/multiple-associated-values-fails.rs:1:10
  |
1 | #[derive(nldl::attr::Serialize)]
  |          ^^^^^^^^^^^^^^^^^^^^^
  |
  = help: message: Failed to parse enum variants in NetlinkAttributeSerializable derive: Variant "Flags" contains more than one associated value.
